apps/public/src/app/(map)/locatie/kies/_components/BaseLayer.tsx

```diff
@@ -1,14 +1,12 @@
-import { Button, Paragraph } from '@amsterdam/design-system-react'
 import L from 'leaflet'
 import { useEffect, useRef, useState } from 'react'
 
 import 'leaflet/dist/leaflet.css'
 
 import type { Coordinates } from '../page'
 
+import { Controls } from './Controls/Controls'
```

Comment:
Nice refactor! We could go a bit further I think. This file is called `BaseLayer`, but it's the base layer + the overlay. Maybe we should just call is `Map.tsx`? And the css `Map.module.css`?

And do you think it's cleaner if we move setting the `mapInstance` to a hook in a separate file? `useAddMapInstance` or something?

Reply:
I renamed the file. The config for the map we can move, but maybe not for this PR.